resources/views/frontend/layouts/partials/menu.blade.php

```diff
@@ -0,0 +1,155 @@
+<div class="body body2">
+  <div id="page">
+    <div id="header" class="headerwrap">
+      <div class="headert">
+        <div class="hd_bg_">
+          <script type="text/javascript">
+            (function($){
+              $(function(){
+                $('.parallax-layer', '#parallax').parallax({
+                  mouseport: $('body')}
+                                                          );
+              }
+               );
+            }
+            )(jQuery);
+          </script>
+          <div class="parallax-viewport" id="parallax">
+            <!-- parallax layers -->
+            <div class="parallax-layer" class="layer1" >
+              <img src="{{asset('{{asset('frontend/imgs/hd_bg.jpg')}}" alt=""  id="img1" style=";" />
+            </div>
+            <div class="parallax-layer" class="layer2" >
+              <img src="{{asset('frontend/imgs/hd_may1.png')}}" alt="" id="img2"  />
```

Comment:
[rank A] will you using id attribute for selector? Please avoid named it clearly more
please don't use any variable or attribute name like abc1, abc2, abc3, ....